frontend/components/ui/blocknote/blocks/quiz-true-false-block.tsx

```
@@ -0,0 +1,183 @@
+          }
+        }, 0);
+      };
+
+      const handleDelete = () => {
+        props.editor.removeBlocks([props.block]);
+      };
+
+      return (
+        <div
+          ref={blockRef}
+          onMouseDown={(e) => {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)